src/node.rs

```diff
@@ -251,14 +251,20 @@ impl Doc {
         author_id: Arc<AuthorId>,
         key: Vec<u8>,
         value: Vec<u8>,
-    ) -> Result<Arc<Hash>, Error> {
+    ) -> Result<Arc<Entry>, Error> {
         block_on(&self.rt, async {
-            let hash = self
-                .inner
-                .set_bytes(author_id.0.clone(), key, value)
+            // TODO - set_bytes should return an entry? probably.
```

Comment:
cc @Frando 

I think we should unifiy the API around either `Hash` or `Entry`. And we need a method to go from `Hash` to `Entry`, currently things are a little bit wonky here